recipe-client-addon/lib/EventEmitter.jsm

```diff
@@ -1,17 +1,16 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-// This file is meant to run inside action sandboxes
-
 "use strict";
 
+const {utils: Cu} = Components;
+Cu.import("resource://shield-recipe-client/lib/LogManager.jsm");
```

Comment:
It's from https://github.com/mozilla/normandy/commit/388cbecf2c8534fea0c9cd8786449c3891ceee08, which I believe got merged before we decided to do reviews on Github and bulk-commit changes to mozilla-central, but we didn't make the bug for it in time before that decision was made. So it'll have to get your review, at which point you'd probably note something like how I missed adding it to the list of modules to unload. :P

If you want, we can just do comments directly on that commit and I'll make a PR with feedback fixes. How does that sound?